scanner/decisions/0014-epss-score.md

```diff
@@ -0,0 +1,128 @@
+
+The [`claircore.EPSS` enricher](https://github.com/quay/claircore/blob/main/enricher/epss/epss.go) is used for data fetching, parsing and enriching, as a component of Scanner V4.
+
+All workflows in GitHub Actions, including those running cron jobs, operate in UTC. Similarly, the EPSS data example aligns with UTC, as shown in the timestamp: `score_date:2025-01-10T00:00:00+0000`. 
+All EPSS data integrated into Scanner V4 corresponds to the previous day in UTC. This approach minimizes the risk of failures caused by fetching data for the current UTC date, which may not yet be available.
+To include the EPSS details in Scanner V4, the protobuf message will be extended to:
+
+```
+message VulnerabilityReport {
+  message Vulnerability {
+    enum Severity {
+      SEVERITY_UNSPECIFIED = 0;
+      ...
+      SEVERITY_CRITICAL = 4;
+    }
+    message CVSS {
+      enum Source {
+        SOURCE_UNKNOWN = 0;
+        ...
+      }
+      message V2 {
+        float base_score = 1;
+        string vector = 2;
+      }
+      message V3 {
+        float base_score = 1;
+        string vector = 2;
+      }
+      V2 v2 = 1;
+      V3 v3 = 2;
+      Source source = 3;
+      string url = 4;
+    }
+    message EPSS { <-- new proto
+      string date = 1;
+      string model_version = 2;
+      float probability = 3;
+      float percentile = 4;
+    }
+    string id = 1;
+    string name = 2;
+    string description = 3;
+    google.protobuf.Timestamp issued = 4;
+    string link = 5 [deprecated = true]; // link is duplicated with CVSS URL field, the exact deprecation date is undecided
+    ...
+    repeated CVSS cvss_metrics = 13;
+    EPSS epss = 14; <-- new field
+  }
+  ...
+  repeated Note notes = 5;
+}
+```
+### Handling RHSA/RHBA/RHEA
+
+Without loss of generality, RHSA/RHBA/RHEA will just be referred to as the more well-known RHSA variant of the three.
+Scanner V4 currently displays RHSA as the top-level entity rather than the related CVE(s) when the CVE(s) are fixed. 
+Meanwhile, all EPSS data are CVE-centric. In Scanner V4, the EPSS score for an RHSA will be the highest EPSS score among all CVEs linked to that RHSA, as multiple CVEs can be associated with a single RHSA.
+This approach matches our current scheme for assigning an RHSA a CVSS score
+
+## Consequences
+
+* We would need to ensure the [First organization's data source](https://epss.cyentia.com/epss_scores-YYYY-MM-DD.csv.gz) remains consistently accessible. 
+Additionally, we should be prepared to change the data updating process if any API key requirements or request throttling are introduced in the future.
+
+* As noted earlier, the EPSS score displayed in Scanner V4 for an RHSA will correspond to the highest EPSS score among the CVEs linked to that RHSA.
+This may change when Scanner V4 introduces CVEs as top-level entities, allowing each CVE to have its own EPSS score, if applicable.
```

Comment:
When Scanner V4 introduces CVEs as top-level entities, we can have each CVE to have its own EPSS score, if applicable. So the RHSA EPSS score calculation will not longer needed.